Resources/Prototypes/Entities/Clothing/Neck/scarfs.yml

```
@@ -1,5 +1,12 @@
 - type: entity
   parent: ClothingNeckBase
+  id: ClothingNeckScarfBase
+  components:
+  - type: TemperatureProtection
+    coefficient: 0.8 # equivalent to winter boots
+    
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)